src/chapters/orm/studio.rst

```diff
@@ -0,0 +1,114 @@
+Studio: Creating an ``AbstractEntity``
+======================================
+
+In :ref:`orm1-exercises`, you created a new persistent class, ``EventCategory``. We will eventually use this class to categorize events in our application. However, your application now has a fair amount of duplication across the two model classes. This studio has you reduce repetition (i.e. "DRY out your code") using inheritance and abstract classes.
```

Comment:
How about, "In the exercises..."?

Also, to put the students in the driver's seat, "In this studio, you will reduce repetition..."

Reply:
This title of the exercises page will change to something like "Exercises: Fun with ORM," so "In Exercises: ..." is probably better in the long run. Sally has this change in her branch.